### PR TITLE
PHP 8 Compatibility fixes

### PR DESCRIPTION
**Pull request by nelsonec87**
## Proposed changes:

These are minor changes as part of our PHP 8.1 compatibility update initiative pdqkMK-UN-p2

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

* pdqkMK-UN-p2
* #23695 
## Testing instructions:

These are minor enough that I think visual inspection should be sufficient but please let me know if you need more information.

### CODE REVIEW

**Comment by nelsonec87** [comment]
Rebased as check was failing

File: projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php [hunk 1]
```diff
@@ -434,8 +434,9 @@ public function render_block_row( $posts, $block_attributes ) {
 	 * @return string
 	 */
 	public function render_block( $attributes ) {
-		$post_id          = get_the_ID();
-		$block_attributes = array(
+		$wrapper_attributes = [];
```
Comment (markbiek) [inline projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php]:
```suggestion
		$wrapper_attributes = array();
```

**Review by markbiek — DISMISSED** [review]
LGTM!

**Comment by nelsonec87** [comment]
Rebased because check was failing

**Review by markbiek — APPROVED** [review]
LGTM!

**Pull request merged**